inc/class_session.php

```diff
@@ -186,11 +186,7 @@ function load_user($uid, $loginkey='')
 			$db->shutdown_query("UPDATE ".TABLE_PREFIX."users SET lastactive='$time', timeonline=timeonline+$timespent{$lastip_add} WHERE uid='{$mybb->user['uid']}'");
 		}
 
-		// Sort out the language and forum preferences.
-		if($mybb->user['language'] && $lang->language_exists($mybb->user['language']))
-		{
-			$mybb->settings['bblanguage'] = $mybb->user['language'];
-		}
+		// Choose date format.
```

Comment:
Maybe we can still have the existence check here.

Reply:
I don't think it is necessary as it is done further down:
https://github.com/mybb/mybb/pull/4046/files#diff-73b6280088a2b814ef3608849f74d75eR592